lib/session.d.ts

```diff
@@ -1,7 +1,8 @@
 declare module 'telegraf-session-local' {
   import { AdapterSync, AdapterAsync, BaseAdapter } from 'lowdb'
   import { Context } from 'telegraf'
-  import { MiddlewareFn } from 'telegraf/typings/composer'
+
+  type MiddlewareFn<C extends Context> = (ctx: C, next: () => Promise<void>) => Promise<void>
```

Comment:
I meant
```suggestion
  type MiddlewareFn<C extends Context> = (ctx: C, next: () => Promise<void>) => Promise<unknown>
```

`Promise<void>` should work too, but `void`'s interactions are inconsistent.

Reply:
void is more precise than unknown?